test/search.cc

```diff
@@ -326,27 +375,47 @@ TEST(Search, test_edge_search) {
   // TODO: add more tests
 }
 
-TEST(Search, test_reachability_radius) {
+TEST(Search, test_reachability_radius_zero_everything) {
   PointLL ob(b.second.first - .001f, b.second.second - .01f);
-  unsigned int longest = ob.Distance(d.second);
-  unsigned int shortest = ob.Distance(a.second);
 
-  // zero everything should be a single closest result
+  LOGLN_WARN("zero everything should be a single closest result");
   search({ob, Location::StopType::BREAK, 0, 0, 0}, 2, 0);
+}
+
+TEST(Search, test_reachability_radius_high) {
+  PointLL ob(b.second.first - .001f, b.second.second - .01f);
+  unsigned int longest = ob.Distance(d.second);
 
-  // set radius high to get them all
+  LOGLN_WARN("set radius high to get them all");
   search({b.second, Location::StopType::BREAK, 0, 0, longest + 100}, 10, 0);
+}
 
-  // set radius mid to get just some
+TEST(Search, test_reachability_radius_mid) {
+  PointLL ob(b.second.first - .001f, b.second.second - .01f);
+  unsigned int shortest = ob.Distance(a.second);
+
+  LOGLN_WARN("set radius mid to get just some");
   search({b.second, Location::StopType::BREAK, 0, 0, shortest - 100}, 4, 0);
+}
 
-  // set reachability high to see it gets all nodes reachable
-  search({ob, Location::StopType::BREAK, 5, 5, 0}, 2, 4);
+TEST(Search, test_reachability_radius_reachability_high) {
+  PointLL ob(b.second.first - .001f, b.second.second - .001f);
 
-  // set reachability right on to see we arent off by one
+  LOGLN_WARN("set reachability high to see it gets all edges reachable");
+  search({ob, Location::StopType::BREAK, 10, 10, 0}, 2, 4);
+}
+
+TEST(Search, test_reachability_radius_off_by_one) {
+  PointLL ob(b.second.first - .001f, b.second.second - .01f);
+
+  LOGLN_WARN("set reachability right on to see we arent off by one");
   search({ob, Location::StopType::BREAK, 4, 4, 0}, 2, 4);
+}
+
+TEST(Search, test_reachability_radius_give_up_early) {
```

Comment:
Do these test cases exercise reach in the case restrictions?

Reply:
no, there are no restrictions in this test suite